source/_integrations/analytics.markdown

```diff
@@ -0,0 +1,143 @@
+
+```yaml
+analytics:
+```
+
+## Basic analytics
+
+This includes:
+
+- The UUID of your system
+- The version of your system
+- The installation type of your system
+
+If your system include the Supervisor this will also contain:
+
+- A boolean that indicates if the system is supported
+- A boolean that indicates if the system is healthy
+
+{% details Example payload %}
+
+```json
+{
+  "huuid": "12a3456bc78d90123ef4567g789012h3",
+  "version": "2021.4.0",
+  "installation_type": "Home Assistant OS",
+  "supervisor": {
+    "healthy": true,
+    "supported": true
+  }
+}
+```
+
+{% enddetails %}
+
+## Usage analytics
+
+You need to enable [basic analytics](#basic-analytics) to be able to enable this.
+
+This includes:
+
+- The names of all your integrations
+
+If your system include the Supervisor this will also contain:
+
+- The name of all installed add-ons
+- The version of all installed add-ons
+- The state of protection mode of all installed add-ons
+- The state of the auto update toggle of all installed add-ons
+
+{% details Example payload %}
+
+```json
+{
+  "huuid": "12a3456bc78d90123ef4567g789012h3",
+  "version": "2021.4.0",
+  "installation_type": "Home Assistant OS",
+  "supervisor": {
+    "healthy": true,
+    "supported": true
+  },
+  "integrations": ["awesome_integration"],
+    "addons": [
+        {
+            "slug": "awesome_addon",
+            "protected": true,
+            "version": "1.0.0",
+            "auto_update": false
+        }
+    ]
+}
+```
+
+{% enddetails %}
+
+## Statistics
+
+You need to enable [basic analytics](#basic-analytics) to be able to enable this.
+
+This includes:
+
+- Number of integrations
+- Number of users
+- Number of entities
+- Number of automations
+
+If your system include the Supervisor this will also contain:
+
+- Number of installed add-ons
+
+{% details Example payload %}
+
+```json
+{
+  "huuid": "12a3456bc78d90123ef4567g789012h3",
+  "version": "2021.4.0",
+  "installation_type": "Home Assistant OS",
+  "supervisor": {
+    "healthy": true,
+    "supported": true
+  },
+  "state_count": 1,
+  "automation_count": 2,
+  "integration_count": 3,
+  "addon_count": 4,
+  "user_count": 5
+}
+```
+
+{% enddetails %}
+
+## Diagnostics
+
+This is not sent to the same [Receiver](#receiver) as the rest of the information enabled with this integration. Crash reports are sent to [Sentry](https://sentry.io/welcome/), this data will not be made public and will only be available to a few core developers.
+
+For now, this is only being done for events inside the Supervisor if your system have that. If this changes to include other parts of the ecosystem that will be mentioned in the releasenotes for the version that changes that and the documentation here will be updated.
```

Comment:
```suggestion
For now, this is only being done for events inside the Supervisor if your system has that. If this changes to include other parts of the ecosystem it will be mentioned in the release notes and the documentation here will be updated.
```